butterfly-slack/src/main/resources/postAbort.json

```diff
@@ -0,0 +1,22 @@
+{
+  "channel": "%s",
+  "text": "Your Butterfly transformation failed",
+  "attachments": [
+    {
+      "text": "Error message goes here",
+      "color": "#cb42f4",
+      "actions": [
+        {
+          "text": "Transformation Details",
+          "type": "button",
+          "url": "https://butterfly.pp-devcos-butterfly.us-central1.gcp.dev.paypalinc.com/transformations/3575",
```

Comment:
Interpolate this with the correct link (I know you already plan to do it, but I am just commenting so we keep track of this pending change)